app/api/feedback/route.tsx

```diff
@@ -13,7 +13,9 @@ export const POST = async (request: NextRequest) => {
     const origin = request.headers.get("origin") || "";
     const page = referer.replace(origin, "");
     const ipAddress = request.headers.get("x-forwarded-for") || "";
-    const userAgentParser = new UAParser(request.headers.get("user-agent"));
+    const userAgentParser = new UAParser(
+      request.headers.get("user-agent") ?? undefined
```

Comment:
Is this because `.get("user-agent")` can be an empty string? So you should make it undefined by default?

Reply:
I think it's that `.get("user-agent")` can be `null` (not `""`) and UAParser can only handle `undefined`